### DISCUSSION TITLE
v3: Frage: Wo wird LightColorState benötigt?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#996] [Q&A]
Bei LightColorBody und LightColorHue kann man ja jetzt "Datenpunktwert anzeigen" auswählen.
Wofür braucht man noch LightColorState?
Funktion identisch?

**Comment by Zefau** [comment]
LightColorState hatte ich mit v3 gelöscht und wegen irgendwelchen Issues wieder hervor gekramt. Es gibt Unterschiede aber keine nennenswerten.

**Reply by mcuiobroker** [reply]
Kann man dann weiter beobachten, ob man es noch braucht. War mir nur aufgefallen bei den Tests.
